tests/testthat/test-clean_redcap.R

```diff
@@ -45,8 +45,8 @@ test_that("extract_nonrepeat_table tibble contains expected columns and data typ
   expect_character(out$text)
   expect_character(out$note)
   expect_double(out$calculated)
-  expect_double(out$dropdown_single)
-  expect_double(out$radio_single)
+  expect_character(out$dropdown_single)
```

Comment:
Assuming this is for `label` - shouldn't `dropdown_single` and `radio_single` be a `factor` rather than `character`?

Reply:
Yes, I'll look into this tomorrow.

Reply:
Actually I think this is correct here, I changed the content of the questions in the sample REDCap to have non-numeric labels and raw data (both sides of the comma), and `clean_redcap` does not enforce any label manipulation or cross-referencing with the metadata. That's currently done at the top level `read_redcap_tidy` separate from `clean_redcap`. So here I just needed to update what we expect to come out of REDCap itself, but reordering of things might be necessary.